src/pkg/services/m365/api/events.go

```diff
@@ -679,3 +679,17 @@ func EventFromMap(ev map[string]any) (models.Eventable, error) {
 
 	return body, nil
 }
+
+func eventCollisionKeyProps() []string {
+	return idAnd("subject")
+}
+
+// EventCollisionKey constructs a key from the eventable's creation time, subject, and organizer.
+// collision keys are used to identify duplicate item conflicts for handling advanced restoration config.
+func EventCollisionKey(item models.Eventable) string {
+	if item == nil {
+		return ""
+	}
+
+	return ptr.Val(item.GetSubject())
```

Comment:
does this need updated to use only the ID for the moment?

Reply:
It's a stub for the moment, and is a long ways from being used.  Won't affect anything either way.